lib/index.ts

```diff
@@ -34,7 +34,7 @@ function b64DecodeUnicode(str: string) {
   );
 }
 
-export function base64UrlDecode(str: string) {
+function base64UrlDecode(str: string) {
```

Comment:
Removed the export here, as it was likely an error from moving the code over. We were previously not publically exporting this function, and I think we don't want to do this either.

Reply:
Good catch, thanks!